### PR TITLE
ET-1016 Add navigation to app

### PR DESCRIPTION
**Pull request by SiaExplains**
Ticket => https://www.notion.so/ET-1016-Implement-a-Navigation-Menu-9e14313ea76d40f08defd1ed1fb06795

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -14,16 +14,18 @@ import Footer from './components/Layout/Footer';
...
     <Router>
       <GlobalProvider>
         <div className="et-app">
           <Header />
-          <div className="et-body">
+          <Navigation />
+          <div className="et-body" id="et-body">
             <div className="et-body__center">
-              <Sidebar />
+              {/* <Sidebar /> */}
```
Comment (SheidaMohassesy) [inline src/App.tsx:28]: It seems you forgot the comment here.

File: src/components/Layout/Header.tsx [hunk 2]
```diff
@@ -3,12 +3,12 @@ import React from 'react';
...
   return (
     <div className="et-header">
-      <div className="et-header__center">
+      <div className="et-header__center" id="et-header">
         <div className="et-logo et-hidden-xs">
           KeepCent <span>(1.0.0)</span>
         </div>
-        &nbsp;
-        <input className="et-header-search-bar" placeholder="Search for a transaction..." />
+        {/* &nbsp;
```
Comment (SheidaMohassesy) [inline src/components/Layout/Header.tsx]: We had a search box here now you commented it. If we don't need this feature please remove the code.

**Review by SheidaMohassesy — COMMENTED** [review]
You commented some features and I left comments regarding them but in general that was an impressive work. Well done

File: src/App.tsx (hunk 1 above)
Comment (SiaExplains) [reply src/App.tsx:28]: Let's keep this component comment here to keep in mind that we need a sidebar in future.

File: src/components/Layout/Header.tsx (hunk 2 above)
Comment (SiaExplains) [reply src/components/Layout/Header.tsx]: Thanks for your feedback, I'll remove this one for now until we make a decision about having or not having search-box.

**Review by SheidaMohassesy — APPROVED** [review]
good job

**Pull request merged**